appinfo/info.xml

```diff
@@ -6,7 +6,7 @@
 	<description><![CDATA[Backup now. Restore later.]]></description>
 	<licence>agpl</licence>
 	<author mail="maxence@artificial-owl.com">Maxence Lange</author>
-	<version>23.0.0-alpha1</version>
```

Comment:
Why use 23?
Version is relative to the app, not compatibility?
I would stick to 1.0.0 or something under for first release.

https://semver.org/